doc/3d-snapshot/GetFieldPtr.c

```diff
@@ -22,70 +22,17 @@ GetFieldPtrValidateT(
 )
 {
   /* Validating field f1 */
-  BOOLEAN hasEnoughBytes0 = (uint64_t)(uint32_t)10U <= (InputLength - StartPosition);
-  uint64_t positionAfterT;
-  if (!hasEnoughBytes0)
+  BOOLEAN hasBytes0 = (uint64_t)10U <= (InputLength - StartPosition);
```

Comment:
NB: This is generated code from this PR branch. The while loops are gone in favor of a simple arithmetic check